### ISSUE TITLE
GitLab CI/CD Local Fails

### ISSUE THREAD

**Issue by DanielRTRD** [lorisleiva/laravel-deployer#66]
Error in GitLab CI/CD
```
In Process.php line 223:
                                                                               
  The command "cd /home/gitlab/builds/6ca79a6c/0/AVS/Hermes/vendor/lorisleiva  
  /laravel-deployer/src/strategy/../../.build && (/usr/bin/git clone  --depth  
   1 --recursive -q  /home/gitlab/builds/6ca79a6c/0/AVS/Hermes/vendor/lorisle  
  iva/laravel-deployer/src/strategy/../../.build/releases/1 2>&1)" failed.     
                                                                               
  Exit Code: 128(Invalid exit argument)                                        
                                                                               
  Working directory: /home/gitlab/builds/6ca79a6c/0/AVS/Hermes                 
                                                                               
  Output:                                                                      
  ================                                                             
  fatal: repository '/home/gitlab/builds/6ca79a6c/0/AVS/Hermes/vendor/lorisle  
  iva/laravel-deployer/src/strategy/../../.build/releases/1' does not exist
```

Config file:
```
<?php

return [
    'default' => 'local',
    'strategies' => [
        //
    ],
    'hooks' => [
        // Right before we start deploying.
        'start' => [
            //
        ],
        
        // Code and composer vendors are ready but nothing is built.
        'build' => [
            //
        ],
        
        // Deployment is done but not live yet (before symlink)
        'ready' => [
            'artisan:storage:link',
            'artisan:view:clear',
            'artisan:cache:clear',
            'artisan:config:cache',
            'artisan:migrate',
        ],
        
        // Deployment is done and live
        'done' => [
            'fpm:reload',
        ],
        
        // Deployment succeeded.
        'success' => [
            //
        ],
        
        // Deployment failed.
        'fail' => [
            //
        ],
    ],
    'options' => [
        'application' => env('APP_NAME', 'Laravel'),
    ],
    'hosts' => [
        //
    ],
    'localhost' => [
        'deploy_path' => '/usr/share/nginx/html',
    ],
    'include' => [
        //
    ],
    'custom_deployer_file' => false,

];
```
Any suggestions on how to fix this?

What I want to do is to locally copy the build folder to the root of the nginx folder.

**Comment by ArtemMolotov** [comment]
#18

**Comment by manjurulcis** [comment]
Running into same problem

**Comment by ArtemMolotov** [comment]
#18

**Comment by roydekleijn** [comment]
Any update?

I keep getting:

[localhost] > cd /Users/user/develop/projects/laravel/project/vendor/lorisleiva/laravel-deployer/src/strategy/../../.build && (/usr/local/bin/git clone -b master --recursive  --reference /Users/user/develop/projects/laravel/project/vendor/lorisleiva/laravel-deployer/src/strategy/../../../../.. --dissociate   /Users/user/develop/projects/laravel/project/vendor/lorisleiva/laravel-deployer/src/strategy/../../.build/releases/1 2>&1)
[localhost] < fatal: repository '/Users/user/develop/projects/laravel/project/vendor/lorisleiva/laravel-deployer/src/strategy/../../.build/releases/1' does not exist

**Comment by lorisleiva** [comment]
Hi everyone 👋 

I'm confused what why this is happening. Does this only happens when deploying from a GitLab pipeline with the local strategy?

That is you're not building the assets on separate GitLab jobs but want Deployer to take care of that for you?

**Comment by lorisleiva** [comment]
Closing due to inactivity. Feel free to open a new issue with more elaborated information. Thank you for your understanding.
